W-13153434: Document support for EU Cloud (#45)
mulesoft/docs-code-builder: modules/ROOT/pages/connect-eu-cloud.adoc

= Access Code Builder From EU Cloud

> Anypoint Code Builder is also available for EU control plane users.

Reviewer: ```suggestion
Anypoint Code Builder is available for EU control plane users.
```

> MuleSoft hosts a version of the Anypoint Platform control plane entirely within the EU (Dublin data centers). This enables you to deploy and manage your APIs, applications, and related data entirely within the European Union. This includes data and metadata about your MuleSoft implementation.

Reviewer: ```suggestion
MuleSoft hosts a version of the Anypoint Platform control plane entirely within the EU (Dublin) data centers. This data center enables you to deploy and manage your APIs, applications, and related data entirely within the European Union, including data and metadata about your MuleSoft implementation.
```

> In addition to EU Cloud, deployment targets for MuleSoft managed runtimes are also hosted in the EU data center. Having both the control and runtime planes entirely within the EU enables you to more effectively meet emerging regulatory requirements.

Reviewer: ```suggestion
In addition to EU Cloud, MuleSoft hosts deployment targets for managed runtimes in the EU data center. Having both the control and runtime planes entirely within the EU helps your organization meet emerging regulatory requirements.
```

[NOTE]

> Anypoint Platform account data is not shared between the US and EU Cloud. Even if you have an account in US Cloud, you must create a new account in EU Cloud.

Reviewer: ```suggestion
Anypoint Platform does not share account data between the US and EU Cloud. Even if you have an account in US Cloud, you must create a new account in EU Cloud.
```
